audit/auditlogtextformatter.go

```diff
@@ -19,64 +19,137 @@ package audit
 import (
 	"errors"
 	"fmt"
+	"sort"
 	"strconv"
 	"strings"
 
+	"github.com/jedib0t/go-pretty/v6/table"
+	"github.com/jedib0t/go-pretty/v6/text"
 	"github.com/logrusorgru/aurora"
+	"github.com/shopspring/decimal"
 	. "github.com/sirupsen/logrus"
 	"github.com/sonatype-nexus-community/nancy/types"
 )
 
+var (
+	nine, seven, four decimal.Decimal
+)
+
+func init() {
+	nine, _ = decimal.NewFromString("9")
+	seven, _ = decimal.NewFromString("7")
+	four, _ = decimal.NewFromString("4")
+}
+
 type AuditLogTextFormatter struct {
 	Quiet   *bool
 	NoColor *bool
 }
 
-func logPackage(sb *strings.Builder, noColor bool, quiet bool, idx int, packageCount int, coordinate types.Coordinate) {
-	if !quiet {
-		au := aurora.NewAurora(!noColor)
-		packageLog := "[" + strconv.Itoa(idx) + "/" + strconv.Itoa(packageCount) + "]" +
-			au.Bold(coordinate.Coordinates).String() +
-			au.Gray(20-1, "   No known vulnerabilities against package/version\n").String()
-		sb.WriteString(packageLog)
-	}
+func logPackage(sb *strings.Builder, noColor bool, idx int, packageCount int, coordinate types.Coordinate) {
+	au := aurora.NewAurora(!noColor)
+	sb.WriteString(
+		fmt.Sprintf("[%d/%d] %s\n",
+			idx,
+			packageCount,
+			au.Bold(au.Green(coordinate.Coordinates)).String(),
+		),
+	)
 }
 
 func logInvalidSemVerWarning(sb *strings.Builder, noColor bool, quiet bool, invalidPurls []types.Coordinate) {
 	if !quiet {
```

Comment:
`if loud {` for a separate PR.

Reply:
Yeah, I wanted to keep that out of here (this was already pretty sprawling as it is).